frontend/src/views/container/container/operate/index.vue

```diff
@@ -368,6 +369,7 @@ const acceptParams = (params: DialogProps): void => {
         }
         dialogData.value.rowData.cmdStr = itemCmd.trimEnd();
         let itemEntrypoint = '';
+        dialogData.value.rowData.entrypoint = dialogData.value.rowData?.entrypoint || [];
         for (const item of dialogData.value.rowData.entrypoint) {
             if (item.indexOf(' ') !== -1) {
                 itemEntrypoint += `"${escapeQuotes(item)}" `;
```

Comment:
The provided code snippet has several improvements:

1. **Optional Chaining Usage**: The use of `dialogData.value &&` is unnecessary as it's redundant when used with the logical OR operator (`||`). For example, `dialogData.value.foo || 'default'` can simply be written as `dialogData.value?.foo`. This makes the code cleaner.

2. **String Concatenation Optimization**:
   ```typescript
   itemCmd += `"${escapeQuotes(item)}" ";
               ```
   Consider using template literals for better readability and performance:
   ```typescript
   itemCmd += `\`${escapeQuotes(item)}\``;
   ```

3. **Trimming Command String**:
   You already handle trimming at the end of `itemCmd`, but consider adding a check to ensure that `itemCmd` or `itemEntrypoint` isn't empty before converting them to strings:
   ```typescript
   if (trimedItemCmd.length > 0) {
       dialogData.value.rowData.cmdStr = trimmedItemCmd.trimEnd();
   } else {
       dialogData.value.rowData.cmdStr = ''; // Set value explicitly if needed
   }

   if (trimmedItemEntrypoint.length > 0) {
       dialogData.value.rowData.entrypoint = trimmedItemEntrypoint.split(',');
   } else {
       dialogData.value.rowData.entrypoint = []; // Ensure it's an array
   }
   ```

4. **Type Checking**:
   It's good practice to typecheck variables if they might contain unexpected data types. However, in this case, since you're only accessing existing properties on `dialogData.value` and handling non-existing options through optional chaining/undefined checks, these concerns may not arise.

Overall, the code should work correctly without making significant changes beyond these refactoring suggestions. Make sure to validate any additional transformations or operations applied to `rowData` after these fixes.